Makefile

```diff
@@ -13,7 +13,11 @@ ERL_OPTS= -pa ebin/ \
 test_all: test test_apps
 
 test: all
-	@erl $(ERL_OPTS) -noshell -s ar test_coverage -s init stop
+	@erl $(ERL_OPTS) -noshell -s ar test_with_coverage -s init stop
+
+distributed-test: all
```

Comment:
"distributed-test" is misleading to me. How do you feel about calling it "multi-node-test"? Or actually, in this context it should probably be named "test-multi-node".

Reply:
As discussed, joined the two targets into one for convenience. The same for `make session`.

Reply:
✅